modules/ROOT/pages/afm-create-monitor.adoc

```diff
@@ -55,6 +55,13 @@ Beneath the name of the monitor is an indicator that explains when the current r
 
 After the monitor has run, the list item for the test suite will indicate whether the monitored endpoints passed or failed.
 
+[IMPORTANT]
+====
+Tests silently fail when they take more than 120 seconds to run. As a workaround, you can split a long-running test into multiple tests. It is suggested to download the monitor, split the test across multiple monitors in the BAT CLI, run and time each test to be sure that it is under 120 seconds, and then upload the monitors.
```

Comment:
"It is suggested" > "MuleSoft suggests", or just the imperative > "Download the monitor, ..."